main.tf

```diff
@@ -48,3 +48,39 @@ resource "kubectl_manifest" "cluster_issuer" {
 
   depends_on = [kubernetes_namespace.cert_manager, helm_release.cert_manager, time_sleep.wait]
 }
+
+module "certificates" {
+  for_each = { for k, v in var.certificates : k => v }
+  source   = "./modules/_certificate"
+
+  name                  = each.key
+  namespace             = try(each.value.namespace, "cert-manager")
```

Comment:
We should use default value as: **var.namespace_name** . Because if user will override **var.namespace_name**, they get error for this certificate, when they use default namespace for  this certificate.

Reply:
Thanks for your review.

changed by 19f67f1